d2l-table-style.html

```diff
@@ -19,7 +19,7 @@
 		<style include="d2l-colors">
 			:host {
 				background-color: transparent;
-				border-collapse: separate;
+				border-collapse: separate !important;
```

Comment:
done. The reset CSS also goes through Polymer's shady dom, so it should reproduce the bug even if the shadow DOM is off.